source/documentation/deploying_apps/index.md

```diff
@@ -68,6 +74,104 @@ A possible exception to this is if your org is mature and has pre-existing space
 * Instances will be restarted if they [exceed memory limits](managing_apps.html#quotas).
 * Your application should write all its log messages to `STDOUT`/`STDERR`, rather than a log file.
 
+
+## Deploying private apps
+
+You can deploy your apps such that they are not publicly accessible from the internet.
+
+A common use case is that you have two apps to deploy:
+
+- a public app for your users to interact with
+- a private app that the public app needs to connect to without the private app being accessible from the internet
+
+To achieve this, you must:
+
+- specify the private route in the private app's manifest
+- set the private app URL as an [environment variable](/deploying_apps.html#environment-variables) in the public app's manifest
+- create a network policy to allow the apps to connect with each other
+
+The two apps must be in the same [space](/orgs_spaces_users.html#spaces) to be able to connect to each other.
```

Comment:
DIT have just made me aware that this is not actually true - it's possible to connect apps from different spaces if you have permission for both spaces. You can't do it with the CLI though, because it takes app names and those aren't unique across spaces.

Reply:
Think we could probably just remove this line.
```suggestion
```

Reply:
agree and changed